ddtrace/profiling/exporter/pprof.py

```diff
@@ -361,35 +361,15 @@ def export(self, events, start_time_ns, end_time_ns):
                 thread_id, thread_name, frames, nframes, exc_type_name, list(ue_events)
             )
 
-        sample_types = (
-            ("cpu-samples", "count"),
-            ("cpu-time", "nanoseconds"),
-            ("wall-time", "nanoseconds"),
-            ("uncaught-exceptions", "count"),
-            ("lock-acquire", "count"),
-            ("lock-acquire-wait", "nanoseconds"),
-            ("lock-release", "count"),
-            ("lock-release-hold", "nanoseconds"),
-        )
-
-        # Handle StackExceptionSampleEvent
-        if stack.FEATURES["stack-exceptions"]:
```

Comment:
Is this feature flag no longer used? This seems to be the only occurrence. If so, there are a few references that can be cleaned up:

```
tests/profiling/test_main.py
45:        if stack.FEATURES["stack-exceptions"]:

tests/profiling/collector/test_stack.py
204:@pytest.mark.skipif(not stack.FEATURES["stack-exceptions"], reason="Stack exceptions not supported")
229:@pytest.mark.skipif(not stack.FEATURES["stack-exceptions"], reason="Stack exceptions not supported")

tests/profiling/exporter/test_pprof.py
419:        if stack.FEATURES["stack-exceptions"]:
420:            filename = "test-pprof-exporter_tracemalloc+stack-exceptions.txt"
479:    if stack.FEATURES["stack-exceptions"]:

ddtrace/profiling/exporter/pprof.py
376:        if stack.FEATURES["stack-exceptions"]:

ddtrace/profiling/collector/stack.pyx
42:    "stack-exceptions": False,
178:    FEATURES['stack-exceptions'] = True
```

Reply:
It is used by the tests, like your grep shows. ;)

Reply:
Right but there no longer is any usage in the core code. It should be moved to the tests then?

Reply:
No, it's just gonna be harder to move to the tests as it uses Cython macro, and it's likely it'll be re-used later to export profiler capabilities to the backend.